scripts/image-builder/configure.sh

```diff
@@ -1,13 +1,19 @@
 #!/bin/bash
 set -exo pipefail
 
-sudo dnf install -y git osbuild-composer composer-cli \
-    cockpit-composer bash-completion podman genisoimage \
-    createrepo syslinux yum-utils selinux-policy-devel jq wget lorax rpm-build
+pkg_list="git osbuild-composer composer-cli \
+	cockpit-composer bash-completion podman genisoimage \
+	createrepo yum-utils selinux-policy-devel jq wget lorax rpm-build"
+
+if [ $(uname -p) != "aarch64" ]; then
+    pkg_list="${pkg_list} syslinux"
+fi
```

Comment:
We should be able to simply remove syslinux package from the list on all platforms. It'll be pulled in by lorax when necessary.

Reply:
@ggiguash looks like you already removed it.